fighthealthinsurance/common_view_logic.py

```
@@ -1701,14 +1702,19 @@ async def sub_in_appeals(appeal: dict[str, str]) -> dict[str, str]:
-                subs["[Professional Name]"] = (
-                    denial.primary_professional.get_full_name()
+            try:
+                if denial.patient_user is not None:
+                    subs["[Patient Name]"] = denial.patient_user.get_legal_name()
+                if denial and denial.primary_professional is not None:
+                    subs["[Professional Name]"] = (
+                        denial.primary_professional.get_full_name()
+                    )
+                if denial.domain:
+                    subs["[Professional Address]"] = denial.domain.get_address()
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)